pkg/forklift-api/webhooks/mutating-webhook/mutators/ovirt-cert-mutator_test.go

```diff
@@ -0,0 +1,36 @@
+package mutators
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/onsi/gomega"
+)
+
+func TestCertAppending(t *testing.T) {
+	g := gomega.NewGomegaWithT(t)
+
+	providedCa, err := ioutil.ReadFile("completeCerts.pem")
+	if err != nil {
+		g.Fail("Failed to read certificate file")
+	}
```

Comment:
so how about replacing this with `Expect(err).ToNot(HaveOccurred())`?

Reply:
I've used it initially and removed it because the error presented in that way was less nice, but I agree that its more readable and clean that way so switching back.